internal/service/postgres/files/postgresql.conf

```diff
@@ -27,3 +27,8 @@ log_destination = 'stderr'
 logging_collector = on
 log_directory = 'pg_log'
 log_filename = 'postgresql.log'
+
+# TLS
+ssl = on
+ssl_cert_file = '/secrets/tls/tls.crt'
+ssl_key_file = '/secrets/tls/tls.key'
```

Comment:
I havnt poked around with ssl but would it better to follow official example? https://github.com/sclorg/postgresql-container/tree/master/examples/enable-ssl . Looks they are going with `/opt/app-root/src/certs..`

Reply:
we could move them but it really doesn't make a difference since we are able to specify their location in the conf file.  The direct benefit to us is that we get to reuse the code that mounts the secret across all of our servers since they are all currently mounting to /secrets/tls/...

I'd rather just leave it as is.